File: IT/Variabili, operatori, strutture di controllo/Java/DoWhileTest.java
/* file DoWhileTest.java */

public class DoWhileTest {
	public static void main(String [] args) {
		/* inizializza la variabile contatore */

		int c = 5;

		/* esegue una prima volta le istruzioni nel ciclo, quindi itera finché la variabile contatore è minore di 5 */

		do {
			System.out.println(c);
		}
		while (c < 5);

		System.out.println("Fine del ciclo.");
	}
}

File: IT/1/Variabili, operatori, strutture di controllo/C++/doWhileTest.cpp
/* file doWhileTest.cpp */

#include <iostream>

using std::cout;
using std::endl;

int main() 
{
	/* inizializza la variabile contatore */

	int c = 5;

	/* esegue una prima volta le istruzioni nel ciclo, quindi itera finché la variabile contatore è minore di 5 */

	do {
		cout << c << endl;
	}
	while (c < 5);

	cout << "Fine del ciclo." << endl;
}

